test/core/extensions/coreIssuanceOrder.spec.ts

```diff
@@ -520,14 +407,14 @@ contract('CoreIssuanceOrder', accounts => {
     describe('when the relayer fees are zero', async () => {
       before(async () => {
         ABIDecoder.addABI(StandardTokenMock.abi);
-        makerRelayerFee = ether(0);
-        takerRelayerFee = ether(0);
+        issuanceOrderMakerRelayerFee = ether(0);
```

Comment:
Is this the same as using the constant `ZERO`?

Reply:
Yes, +1